src/Scripting/CSharpTest.Desktop/CsiTests.cs

```diff
@@ -47,7 +48,17 @@ public void CurrentWorkingDirectory_Change()
 new C()
 Environment.Exit(0)
 ";
-            var result = ProcessUtilities.Run(CsiPath, "", stdInput: stdInput);
+            ProcessResult result = null;
```

Comment:
Interesting that we have only seen this issue with this test. Seems like all tests redirecting stdin would have the same issue. I looked into the Process.Start impl and indeed it calls Console.InputEncoding:

```C#
if (startInfo.RedirectStandardInput) {
                standardInput = new StreamWriter(new FileStream(standardInputWritePipeHandle, FileAccess.Write, 4096, false), Console.InputEncoding, 4096);
                standardInput.AutoFlush = true;
            }
```

Let's move this code to ProcessUtilities.Run helper (around Process.Start()).